R/arc-select.R

```diff
@@ -18,50 +20,20 @@ arc_select <- function(
     filter_geom,
     predicate = "intersects",
     n_max = Inf,
-    ...
+    ...,
+    error_call = rlang::caller_env()
```

Comment:
I believe `rlang::caller_env()` does not need to be inlcuded as an argument. If so, the env will be where arc_select() is called which is likely the global env. Can we remove this and then add `call = rlang::caller_env()` in the cli_abort() call?

Reply:
The reason I'd like to keep it here is because I build a lot of functions around ArcGIS endpoints and if it takes a call parameter, it is easier to trace the error back to my function (or just show that name in the default error results) rather than kicking back an error with arc_select as the caller. I'm not stuck on it but I do think it would be helpful.

Reply:
Good argument. I'll make the change.